scalafmt-tests/src/test/resources/rewrite/RedundantBraces-if.stat

```diff
@@ -249,3 +249,41 @@ object a {
   } else if (d)
     println("d!")
 }
+<<< don't rewrite on disabled flag
+rewrite.redundantBraces.ifElseExpressions = false
```

Comment:
should we add `generalExpressions = true` here as well? so that the test verifies that if-else is controlled via the new flag and not the old one...

Reply:
good idea